static/app/views/performance/transactionSummary/transactionOverview/content.spec.tsx

```diff
@@ -143,66 +143,21 @@ describe('Transaction Summary Content', function () {
         onChangeFilter={() => {}}
         router={router}
       />,
-      routerContext
+      {context: routerContext}
     );
 
-    await tick();
-    wrapper.update();
+    screen.debug();
 
-    expect(wrapper.find('Filter')).toHaveLength(1);
-    expect(wrapper.find('SearchBar')).toHaveLength(1);
-    expect(wrapper.find('TransactionSummaryCharts')).toHaveLength(1);
-    expect(wrapper.find('TransactionsList')).toHaveLength(1);
-    expect(wrapper.find('UserStats')).toHaveLength(1);
-    expect(wrapper.find('StatusBreakdown')).toHaveLength(1);
-    expect(wrapper.find('SidebarCharts')).toHaveLength(1);
-    expect(wrapper.find('DiscoverQuery')).toHaveLength(2);
+    expect(screen.getByTestId('page-filter-environment-selector')).toBeInTheDocument();
+    expect(screen.getByTestId('page-filter-timerange-selector')).toBeInTheDocument();
+    expect(screen.getByTestId('smart-search-bar')).toBeInTheDocument();
+    expect(screen.getByTestId('transaction-summary-charts')).toBeInTheDocument();
+    expect(screen.getByRole('heading', {name: /user misery/i})).toBeInTheDocument();
 
-    const transactionListProps = wrapper.find('TransactionsList').first().props();
-    expect(transactionListProps.generateDiscoverEventView).toBeUndefined();
-    expect(transactionListProps.handleOpenInDiscoverClick).toBeUndefined();
-    expect(transactionListProps.generatePerformanceTransactionEventsView).toBeDefined();
-    expect(transactionListProps.handleOpenAllEventsClick).toBeDefined();
-  });
-
-  it('Renders TransactionSummaryCharts withoutZerofill', async function () {
```

Comment:
I opted to delete this second test because it's only performing an equality check on a specific prop. There is no obvious way to confirm the effect of this prop in the actual view in RTL